boot/freeldr/freeldr/arch/uefi/uefimem.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * PROJECT:     Freeldr UEFI Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     UEFI Memory Managment Functions
+ * COPYRIGHT:   Copyright 2022 Justin Miller <justinmiller100@gmail.com>
+ */
+
+/**** Includes ***********************************************************************************/
```

Comment:
You padded here up to column 100.
/**** Includes ***********************************************************************************/
I would recommend to pad only up to column 80 maximum, and leave only one or two * before the word, like:
/** INCLUDES *****************************************************************/

Same for the other 3 separator lines in here.

Reply:
> and leave only one or two * before the word

Just do as we do elsewhere in our codebase.